clang/lib/Driver/ToolChains/PS4CPU.cpp

```diff
@@ -359,6 +359,12 @@ void toolchains::PS4PS5Base::addClangTargetOptions(
 
   CC1Args.push_back("-fno-use-init-array");
 
+  // Default to -fglobal-new-delete-visibility=source for PS5.
```

Comment:
Spelling in comment needs to match the option spelling.

Reply:
Thanks - now using the correct spelling.